docs/feature_adc_driver.md

```diff
@@ -0,0 +1,42 @@
+# ADC Driver
+
+QMK can read analog values on certain pins. For AVR MCUs, these values are 10-bit (0-1023) and mapped between 0V and VCC (5V or 3.3V). This can be useful for implementing things like volume controls or battery level indicators.
+
+## Usage
+
+To use this driver, simply add the following to the top of your code:
+
+```c
+#include "analog.h"
```

Comment:
I don't think that this is quite accurate. 

I don't think that `analog.c` is added to `SRC` at any point, so you'd need to either include the c file, or include the header, and add analog.c to SRC.

Reply:
Yeah, I recall having to do that in lufa.mk to get the Adafruit BLE battery reading working.